demos/android/MASVS-CRYPTO/MASTG-DEMO-0015/run.sh

```diff
@@ -0,0 +1 @@
+ semgrep -c ../rules/mastg-android-hardcoded-crypto-keys-usage.yml ./MastgTest_reversed.java --text -o output.txt  
```

Comment:
Please always test if this works. It wasn't able to find the rule when I ran it.

```suggestion
NO_COLOR=true semgrep -c ../../../../rules/mastg-android-hardcoded-crypto-keys-usage.yml ./MastgTest_reversed.java --text -o output.txt
```